contribs/gmf/src/directives/disclaimer.js

```diff
@@ -189,7 +194,9 @@ gmf.DisclaimerController.prototype.$onInit = function() {
 gmf.DisclaimerController.prototype.handleLayersAdd_ = function(evt) {
   const layer = evt.element;
```

Comment:
`const layer = /** @type {ol.layer.Base} */ (evt.element);`

With that the code compiles for me.

Reply:
strange that the `goog.asserts.assertInstanceof` bellow doesn't work

Reply:
Yes, that's because it's outside the timeout. I've moved it in. It should compile now.
(The @pfirpfel solution should work too)